js-intermediate.js

```diff
@@ -62,10 +95,29 @@ var middleLetters1 = "hello"
 var middleLetters2 = "rhinoceros"
 // Expected output: “oc”
 
+console.log(middleLetters1 [Math.round((middleLetters1.length-1)/2)])
+console.log(middleLetters2[Math.floor((middleLetters2.length - 1) / 2)] + (middleLetters2[Math.round((middleLetters2.length - 1) / 2)]))
+const middleL = (string) => {
+  return string[Math.round((string.length - 1) / 2)]
+}
 
+console.log(middleL(middleLetters1))
+console.log(middleL(middleLetters2))
 
 
 // 7. Write a program to get the area of a sphere using object classes. Create three spheres with different radi as test cases. Area of a sphere =  4πr^2 (four pi r squared)
+var r1 = 2
+var r2 = 5
+var r3 = 100
+
+const areaSphere = (radi) => {
+  let sphere = (4 * Math.PI * (radi*radi))
+  return sphere
+}
+
```

Comment:
The goal of this problem is to create a class called Sphere and then create three objects that are instances of the class.